test/promql/tests/data_plane/availability_metric_integration_service.yaml

```
@@ -0,0 +1,66 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.availability_metric_integration_service_alerts.yaml
+
+tests:
+  # Test 1: Availability drops below 99% for more than 10 minutes on cluster01, triggering an alert.
+  - interval: 1m
+    input_series:
+      - series: 'redhat_appstudio_integrationservice_global_github_app_available{source_cluster="cluster01"}'
+        values: '1x50 0x70'  # 50 minutes of uptime, followed by 70 minutes of downtime
+      - series: 'redhat_appstudio_integrationservice_global_github_app_available{source_cluster="cluster02"}'
+        values: '1x120'  # Cluster02 remains up for the entire period.
+
+    alert_rule_test:
+      - eval_time: 60m  
```

I believe yaml-lint complains about trailing spaces on this line
```suggestion
      - eval_time: 60m
```